pages/project.py

```diff
@@ -117,21 +120,46 @@ class ForksPage(GuidBasePage):
 
 
 class FilesPage(GuidBasePage):
-    base_url = settings.OSF_HOME + '/{guid}/files/'
-
-    identity = Locator(By.CSS_SELECTOR, '#treeGrid')
+    base_url = settings.OSF_HOME + '/{guid}/files/{addon_provider}'
+
+    def __init__(
+        self, driver, verify=False, guid='', addon_provider='', domain=settings.OSF_HOME
+    ):
+        super().__init__(driver, verify)
+        self.guid = guid
+        self.addon_provider = addon_provider
+
+    @property
+    def url(self):
+        if '{guid}' in self.base_url and '{addon_provider}' in self.base_url:
```

Comment:
Might want to delimit these. `s3` could be part of a project guid that doesn't have `s3` attached.

Reply:
Wait, ignore this.  I just got this line and I'm an idiot.